adi/ad4020.py

```diff
@@ -45,6 +55,38 @@ def sampling_frequency(self, value):
         """Set the sampling frequency."""
         self._set_iio_dev_attr("sampling_frequency", str(value))
 
+    class _channel_adc(attribute):
+        """AD4000 series differential input voltage channel"""
+
+        # AD4000 series ADC channel
+        def __init__(self, ctrl, channel_name, output):
+            self.name = channel_name
+            self._ctrl = ctrl
+            self._output = output
+
+        @property
+        def raw(self):
+            return self._get_iio_attr(self.name, "raw", self._output)
+
+        @property
+        def scale(self):
+            return float(self._get_iio_attr_str(self.name, "scale", self._output))
+
+        @scale.setter
+        def scale(self, value):
+            self._set_iio_attr(self.name, "scale", False, str(Decimal(value).real))
+
+        @property
+        def scale_available(self):
```

Comment:
@machschmitt I believe these are all fixed-scale devices. Accessing the scale_available property throws a KeyError.